ibm_cloud_sdk_core/api_exception.py

```diff
@@ -13,17 +13,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import requests
```

Comment:
you can import just the `Response`, just like below for `Optional`

Reply:
👍